### DISCUSSION TITLE
Session history table

### DISCUSSION THREAD

**Discussion by pasqualeceglie** [olahallengren/sql-server-maintenance-solution#458] [Ideas]
Hi Ola, I was thinking about a table (named SessionLog) to store every execution with a sessionid (fk on CommandLog), StartDateTime, EndDateTime and an xml field with parameter list and values (with which the procedure was executed).
What do you think about this?

Best regards.

**Comment by olahallengren** [comment]
@pasqualeceglie, this is a good idea. I will consider this for a future version.
